fx/inline_function.py

```diff
@@ -40,6 +40,7 @@ def forward(self, x):
 # represented as a `call_module` Node. The full operation in the
 # generated `forward` function's code will appear as `self.relu(x)`
 m = symbolic_trace(M())
+tracer = torch.fx.proxy.GraphAppendingTracer(m.graph)
```

Comment:
nit: IMO this should appear after the comment on line 46, since it is logically part of the modification process that starts there rather than the initial program capture up here

Reply:
Resolved in b9071ca below.